util/packaging/rpm/recap.spec

```diff
@@ -47,9 +47,26 @@ DESTDIR=%{buildroot} make install
 %{_mandir}/man5/recap.5.gz
 %{_mandir}/man8/recap.8.gz
 %{_mandir}/man8/recaplog.8.gz
+%{_mandir}/man8/recaptool.8.gz
 
 
 %changelog
+* Mon May 15 2017 Tony Garcia <tony.garcia@rackspace.com> - 1.0.0-1.rs
+- Record point-in-time CPU.
+- Multiple bug fixing on recaptool.
+- Multiple bug fixing on recap.
+- Refactor of recaplog.
+- Fix plesk mysql bug.
+- Install recaptool man page.
+- Adding specific requirements for bash(>=4) and sysstat(>=9).
+- Removing rpm spec requirements: elinks.
+- Obsoletes and provides rs-sysmon.
+- Better support for multiple mysql instances.
+- Makefile updated to allow multiple distros alignment and to include new man pages.
+- Adjust spec for Makefile.
+- Makefile now uses /usr/local as default for DESTDIR.
+- fdisk warnings(stderr) are now included in the logs(stdin).
+
```

Comment:
The goal of the RPM changelog isn't to duplicate the upstream changelog, but rather point out packaging changes.  This entry should just be shortened to:

```
- Latest version
- Include recaptool man page
- Drop elinks requirement
- Add psmisc requirement
- Require minimum versions of bash (4) and sysstat (9)
```

You could even combine the last three into a generic `- Clean up requirements` entry.